Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2_test.py

```diff
@@ -1929,3 +1929,26 @@ def test_informative_error_in_get_token(mocker):
         Client('test_client')
 
     assert str(err.value) == 'Unauthorized - Incorrect user id or password'
+
+
+def test_headers_file_acquisition_package_request(requests_mock, mocker):
+    """
+    Given:
+        - mock client, acquisition_id
+    When:
+        - running the file_acquisition_package_request
+    Then:
+        - ensure that the headers of this command is what expected.
```

Comment:
Let's mention the expected headers explicitly 
```suggestion
        - ensure that the headers of this command is _____
```